soccer/planning/MotionCommand.hpp

```diff
@@ -0,0 +1,74 @@
+#pragma once
+
+#include <Geometry2d/Point.hpp>
+#include <boost/optional.hpp>
+#include "planning/MotionInstant.hpp"
+
+
+namespace Planning {
+/**
+ * This class contains the motion constraints that the high-level logic sets for a robot.
+ * For position: set EITHER @motionTarget OR @targetWorldVel.
+ * For angle: set EITHER @targetAngleVel OR @faceTarget.
+ */
+    class MotionCommand {
```

Comment:
Let's move the angle target stuff into this for now, since it makes more sense here than in MotionConstraints.  We may decide later to have a TransMotionCommand and an AngleMotionCommand.